.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+sudo: false
+cache: bundler
+script:
+  - "bundle exec rspec spec"
```

Comment:
yeah, rake spec is much more conventional ... who even know there was an rspec executable?  ruby--.